internal/fwschema/schema_test.go

```diff
@@ -188,3 +192,359 @@ func TestSchemaBlockPathExpressions(t *testing.T) {
 		})
 	}
 }
+
+func TestSchemaAttributeAtTerraformPath(t *testing.T) {
```

Comment:
Amongst many other great things in this PR, ❤️ for this additional unit testing that was missing before.